docs/features/UNIFIED_CANVAS.md

```diff
@@ -0,0 +1,96 @@
+
+If you desire something completely different than the original image in your new generation (i.e., ignoring existing colors/forms), consider toggling the Inpaint Replace setting on, and use high values for both that and Image to Image strength.
+
+Note: By default, the **Scale Before Processing** option will be set to automatically generate at a larger resolution and scale down to inpaint more coherent details. In order to leverage this feature, the best inpainting results will be found by resizing the bounding box to a smaller area that contains your mask, and updating your prompt to describe *just* the area within the bounding box.
+
+### Outpainting - 
+If the Bounding Box is placed over an area of canvas partially filled by an existing Base Layer area and partially by transparent pixels or masks, invoking a new image will use **Outpainting**, as well as **Inpainting** any masked areas.
+ 
+
+# Advanced Features
+
+Features with non-obvious behavior are detailed below, in order to provide clarity on the intent and common use cases we expect for utilizing them.
+
+## Toolbar
+
+### Mask Options
+* **Enable Mask** - This flag can be used to Enable or Disable the currently painted mask. This allows you the option to maintain your mask layer without it affecting current invocations.
+* **Preserve Masked Area** - When enabled, Preserve Masked Area inverts the inpainting process, maintaining the masked areas and regenerating unmasked areas.
+
+### Creative Tools
+* **Brush - Base/Mask Modes** - Without user intervention, the Brush tool will operate between two modes for Base and Mask Layers respectively. On Base Layer mode, the brush will directly paint on the canvas using the color selected on the Brush Options menu. On Mask Layer mode, the brush will create a new mask. The color of the mask created is controlled by the color selector on the Mask Options dropdown.
+* **Erase Bounding Box** - Erases all content on the base layer within the current bounding box.
+* **Fill Bounding Box** - Fill the base layer within the current bounding box with the currently selected color
+
+### Canvas Tools
+* **Move Tool** - Allows for manipulation of the canvas view (by dragging on the canvas, outside the bounding box), the Bounding Box (using the edges of the box), or the Width/Height of the Bounding Box (using the 9-directional handles).
+* **Reset View** - Will reorient the user to the image within the Bounding box.
+* **Merge Visible** - In the event your computer is demonstrating performance issues, merging visible layers will create a merged copy of the image in order to continue your composition without maintaining the full set of front-end layers in your browser.
+
+## Bounding Box
+* **Generation Area** - The Bounding Box serves as the area used to inform where invocation should occur. It's Width & Height can be manipulated directly via the Bounding Box tab. 
+
+## Seam Correction
```

Comment:
The seam is used for both inpainting and outpainting, is that right? You might want to mention these operations explicitly.